services/core-api/app/api/parties/party_appt/resources/mine_party_appt_resource.py

```diff
@@ -56,13 +56,15 @@ class MinePartyApptResource(Resource, UserMixin):
     parser.add_argument(
         'mine_party_acknowledgement_status',
         type=MinePartyAcknowledgedStatus,
-        choices=list(MinePartyAcknowledgedStatus),
-        help='Indicator of Ministry acknowledgement of the appointment.',
+        choices=list(MinePartyAcknowledgedStatus) + list([None]),
+        help='Indicator of status of acknowledgement.',
+        default=MinePartyAcknowledgedStatus.not_acknowledged,
+        location='json',
         store_missing=False)
     parser.add_argument(
         'status',
         type=MinePartyAppointmentStatus,
-        choices=list(MinePartyAppointmentStatus),
+        choices=list(MinePartyAppointmentStatus) + list([None]),
```

Comment:
I just don't think that a status of None should be valid, if we're getting null status we should change it, not allow it. Acknowledgement of appointments makes sense as None because not every appointment _needs_ to be acknowledged, but every appointment _should_ be in a state of active/inactive/pending. If we're getting null status it should probably be set to active.

Down around line 284 we're only setting status to active if the appointment is acknowledged, assuming that appointments that require acknowledgement are sent over properly as "not_acknowledged", I think the logic should be flipped to value !== not_acknowledged.

Reply:
Conversely, since these aren't required fields and are in fact nullable, we don't want the api to fail if None is passed for whatever reason.

I'm pretty sure both of these fields are only utilized in the TSF roles section.  So, it wouldn't matter to any other assignments whether they are "active" or not.  I'd rather not set values for entries that aren't relevant.